Add new KEB step for external docker registry (#297)
kyma-project/control-plane: docs/kyma-environment-broker/03-03-runtime-operations.md

| Create_LMS_Tenant                      | LMS                      | Requests a tenant in the LMS system or provides a tenant ID if it was created before.                                                              | @piotrmiskiewicz (Team Gopher) |
| IAS_Registration                       | Identity Authentication Service | Registers a new ServiceProvider on IAS, generates client ID and Secret, and inserts them to Grafana overrides. This step is not required and can be disabled. | @jasiu001 (Team Gopher) |
| EDP_Registration                       | Event Data Platform      | Registers an SKR on Event Data Platform with the necessary parameters. This step is not required and can be disabled. | @jasiu001 (Team Gopher) |

> | Provision Azure Resource Group         | Azure                    | Creates the Azure Resource Group that will be used in other Azure related steps.                                                                | @nachtmaar (Team SkyDivingTunas), @tgorgol (Team Wookiee)        |

Reviewer: ```suggestion
| Provision Azure Resource Group         | Azure                    | Creates the Azure Resource Group that is used in other Azure-related steps.                                                                | @nachtmaar (Team SkyDivingTunas), @tgorgol (Team Wookiee)        |
```

| Provision Azure Event Hubs             | Event Hub                | Creates the Azure Event Hub Namespace which is a managed Kafka cluster for a Kyma Runtime.                                                       | @anishj0shi (Team SkyDivingTunas)     |

> | Provision Azure Container Registry     | Serverless               | Creates the Azure Container Registry that will be used by Serverless to store lambda functions' images.                                         | @tgorgol (Team Wookiee)        |

Reviewer: ```suggestion
| Provision Azure Container Registry     | Serverless               | Creates the Azure Container Registry that is used by Serverless to store images of lambda Functions.                                         | @tgorgol (Team Wookiee)        |
```

| Name                         | Domain         | Status      | Description                                                                            | Owner     |
|------------------------------|----------------|-------------|----------------------------------------------------------------------------------------|-----------|
| Deprovision_Initialization   | Deprovisioning | Done        | Initializes the `DeprovisioningOperation` instance with data fetched from the `ProvisioningOperation`. | @polskikiel (Team Gopher) |

> | Deprovision Azure Resource Group | Azure  | In progress | Deletes the Azure Resource Group along with all resources it contains.                                                  | @montaro (Team SkyDivingTunas)   |

Author: Sure, I'll do the same for provisioning of the resource group

> | Deprovision Azure Resource Group | Azure  | In progress | Deletes the Azure Resource Group along with all resources it contains.                                                  | @montaro (Team SkyDivingTunas)   |

Reviewer 2: That is a good question! I have no idea.

> | Deprovision Azure Resource Group | Azure  | In progress | Deletes the Azure Resource Group along with all resources it contains.                                                  | @montaro (Team SkyDivingTunas)   |

Reviewer 3: it means that when the doc was created, the step was not finished, let's change to done

> | Deprovision Azure Resource Group | Azure  | Done | Deletes the Azure Resource Group along with all resources it contains.                                                  | @montaro (Team SkyDivingTunas), @tgorgol (Team Wookiee)      |

Reviewer: ```suggestion
| Deprovision Azure Resource Group | Azure  | Done | Deletes the Azure Resource Group along with all resources it contains, such as Azure Container Registry and/or Azure Event Hubs.                                                  | @montaro (Team SkyDivingTunas), @tgorgol (Team Wookiee)      |
```
